### PR TITLE
added tailwind fonts and route to products#index

### PR DESCRIPTION
**Pull request by valentin-orrit**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # yarn lockfile v1
 
 
-"@esbuild/darwin-x64@0.21.4":
+"@esbuild/linux-x64@0.21.4":
```
Comment (Nzoops) [inline yarn.lock:5]: Who have raison?

**Review by Nzoops — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (1110lias) [inline yarn.lock:1]: faut choisir les gars!

Comment (1110lias) [reply yarn.lock:1]: Je vote Linux

File: yarn.lock (hunk 1 above)
Comment (valentin-orrit) [reply yarn.lock:5]: me

**Pull request merged**